lib/active_model/serializable_resource.rb

```diff
@@ -11,7 +11,23 @@ def initialize(resource, options = {})
         options.partition { |k, _| ADAPTER_OPTION_KEYS.include? k }.map { |h| Hash[h] }
     end
 
-    delegate :serializable_hash, :as_json, :to_json, to: :adapter
+    def serializable_hash(*args)
+      notify_active_support do
+        adapter.serializable_hash(*args)
+      end
+    end
+
+    def as_json(*args)
+      notify_active_support do
+        adapter.as_json(*args)
+      end
+    end
+
+    def to_json(*args)
+      notify_active_support do
+        adapter.to_json(*args)
+      end
+    end
```

Comment:
Hi @bf4 I updated to use this https://github.com/rails-api/active_model_serializers/commit/49c8f69080f9734a0bda5268aa7ede56f753b465. Personally I don't like, but I can live with it :smile: